lumicks/pylake/kymo.py

```diff
@@ -39,11 +39,14 @@ class Kymo(ConfocalImage):
         End point in the relevant info wave.
     json : dict
         Dictionary containing kymograph-specific metadata.
+    position_offset : float
+        Coordinate position offset.
     """
 
-    def __init__(self, name, file, start, stop, json):
+    def __init__(self, name, file, start, stop, json, position_offset=0):
         super().__init__(name, file, start, stop, json)
         self._line_time_factory = _default_line_time_factory
+        self.position_offset = position_offset
```

Comment:
should this be an internal attribute maybe? at least until we determine if we really need it?

Reply:
Alright. I've made it private for now.